pyramid_nacl_session/serializer.py

```diff
@@ -21,11 +21,13 @@ class EncryptedSerializer(object):
         should be raised for malformed inputs. Default: ``None``, which will
         use :class:`pyramid.session.PickleSerializer`.
     """
+
     def __init__(self, secret, serializer=None):
         if len(secret) != SecretBox.KEY_SIZE:
             raise ValueError(
-                "Secret should be a random bytes string of length %d" %
-                SecretBox.KEY_SIZE)
+                "Secret should be a random bytes string of length %d"
+                % SecretBox.KEY_SIZE
+            )
```

Comment:
Rather than a gratuitous re-wrap, please use the more modern `.format()` syntax instead, e.g.:

```python
                "Secret should be a random bytes string of length {}".format(
                    SecretBox.KEY_SIZE
```